libs/ledger-live-common/src/families/polkadot/polkadot-crypto.ts

```diff
@@ -0,0 +1,11 @@
+import "@polkadot/wasm-crypto/initOnlyAsm";
```

Comment:
I think this is mobile-only, for desktop we might want to take the wasm version instead (which is the default one).

Reply:
oh ok, so possibly we move the **import** back to the reactNative entry then? but still keep the other code?

Reply:
yep I think this would do the trick 👍